src/libraries/Microsoft.Extensions.Logging.EventLog/src/EventLogSettings.cs

```diff
@@ -13,22 +13,22 @@ public class EventLogSettings
         private IEventLog? _eventLog;
 
         /// <summary>
-        /// Name of the event log. If <c>null</c> or not specified, "Application" is the default.
+        /// Gets or sets the name of the event log. If <see langword="null" /> or not specified, "Application" is the default.
         /// </summary>
         public string? LogName { get; set; }
 
         /// <summary>
-        /// Name of the event log source. If <c>null</c> or not specified, ".NET Runtime" is the default.
+        /// Gets or sets the name of the event log source. If <see langword="null" /> or not specified, ".NET Runtime" is the default.
         /// </summary>
         public string? SourceName { get; set; }
 
         /// <summary>
-        /// Name of the machine having the event log. If <c>null</c> or not specified, local machine is the default.
+        /// Gets or sets the name of the machine with the event log. If <see langword="null" /> or not specified, local machine is the default.
```

Comment:
For consistency with the others:

```suggestion
        /// Gets or sets the name of the machine with the event log. If <see langword="null" /> or not specified, "local machine" is the default.
```

@dotnet/area-extensions-logging does this default value begin in lowercase?